build_projects/update-dependencies/Program.cs

```diff
@@ -60,6 +60,20 @@ public static void Main(string[] args)
             }
         }
 
+        private static BuildInfo GetBuildInfo(string name, string buildInfoFragment, bool fetchLatestReleaseFile = true)
```

Comment:
I wanted to retain the signature of Version Tools's BuildInfo.Get.  Logically, this behaves like that method except that it just also handles the case where `DOTNET_VERSION_URL` has a `file://` protocol.  That's why I picked the default that I did.